core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-reports/components/dot-experiments-reports-chart/dot-experiments-reports-chart.component.scss

```diff
@@ -6,3 +6,26 @@
 height='15.7241' rx='7.86207' fill='%23F4F4F6'/%3E%3Crect x='90.5' y='477.725' width='92' height='16.1758' rx='8.08791' fill='%23F4F4F6'/%3E%3Crect x='270.5' y='458' width='72' height='15.7241' rx='7.86207' fill='%23F4F4F6'/%3E%3Crect x='260.5' y='477.725' width='92' height='16.1758' rx='8.08791' fill='%23F4F4F6'/%3E%3Crect x='440.5' y='458' width='72' height='15.7241' rx='7.86207' fill='%23F4F4F6'/%3E%3Crect x='430.5' y='477.725' width='92' height='16.1758' rx='8.08791' fill='%23F4F4F6'/%3E%3Crect x='610.5' y='458' width='72' height='15.7241' rx='7.86207' fill='%23F4F4F6'/%3E%3Crect x='600.5' y='477.725' width='92' height='16.1758' rx='8.08791' fill='%23F4F4F6'/%3E%3Crect x='780.5' y='458' width='72' height='15.7241' rx='7.86207' fill='%23F4F4F6'/%3E%3Crect x='770.5' y='477.725' width='92' height='16.1758' rx='8.08791' fill='%23F4F4F6'/%3E%3C/g%3E%3Cline x1='61.9375' y1='59.4612' x2='31.5003' y2='59.4612' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='17.4612' x2='31.5003' y2='17.4612' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='101.461' x2='31.5003' y2='101.461' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='143.461' x2='31.5003' y2='143.461' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='185.461' x2='31.5003' y2='185.461' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='227.461' x2='31.5003' y2='227.461' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='269.461' x2='31.5003' y2='269.461' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='311.461' x2='31.5003' y2='311.461' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='353.461' x2='31.5003' y2='353.461' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='395.461' x2='31.5003' y2='395.461' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Cline x1='61.9375' y1='437.461' x2='31.5003' y2='437.461' stroke='%23F4F4F6' stroke-width='0.922339'/%3E%3Crect x='0.5' y='10' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='52' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='93' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='136' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='180' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='220' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='261' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='304' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='346' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='386' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Crect x='0.5' y='430' width='31' height='16' rx='8' fill='%23F4F4F6'/%3E%3Cdefs%3E%3CclipPath id='clip0_3061_22368'%3E%3Crect width='855' height='515' fill='white' transform='translate(61.5)'/%3E%3C/clipPath%3E%3C/defs%3E%3C/svg%3E");
```

Comment:
Is there a `material icon` for this?

Reply:
PrimeNG Icons would be better, I think that material icons will be replaced by it

Reply:
> Is there a `material icon` for this?

No is this image:

<img width="934" alt="Screenshot 2023-04-14 at 3 48 15 PM" src="https://user-images.githubusercontent.com/1909643/232141625-cb247cb8-1122-42ab-809c-de25dd21f97c.png">

Reply:
> 

I know. It is a background image.